modules/core/src/v2/coins/trx.ts

```diff
@@ -318,9 +318,9 @@ export class Trx extends BaseCoin {
    * @param callback
    */
   getExtraPrebuildParams(buildParams: any, callback?: NodeCallback<any>): Bluebird<any> {
-    const recipients: any[] = [{}];
+    const recipients: any[] = [{ ...buildParams.recipients[0] }];
```

Comment:
we need to check if this exists before we use a spread operator on it and that it can even be spread